Resources/qc-api/metadata-table.html

```diff
@@ -0,0 +1,113 @@
+<table class="qc-table table">
+  <thead>
+    <tr>
+      <th style="width: 20%">Chart</th>
+      <th style="width: 20%">Series</th>
+      <th style="width: 60%">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td rowspan="2">Strategy Equity</td>
+      <td>Equity</td>
+      <td>Time series of the equity curve</td>
+    </tr>
+    <tr>
+      <td>Daily Performance</td>
+      <td>Time series of daily percentage change</td>
+    </tr>
+    <tr>
+      <td>Capacity</td>
+      <td>Strategy Capacity</td>
+      <td>Time series of <a href="/docs/v2/writing-algorithms/key-concepts/glossary#08-capacity">strategy capacity</a> snapshots</td>
+    </tr>
+    <tr>
+      <td>Drawdown</td>
+      <td>Equity Drawdown</td>
+      <td>Time series of equity peak-to-trough value</td>
+    </tr>
+    <tr>
+      <td>Benchmark</td>
+      <td>Benchmark</td>
+      <td>Time series of the benchmark closing price (SPY, by default)</td>
+    </tr>
+    <tr>
+      <td rowspan="18">Exposure</td>
+      <td>Equity - Long Ratio</td>
```

Comment:
These asset class plots will get outdated when we add new asset classes.
Let's just use a placeholder for all of them
```
    <tr>
      <td rowspan="2">Exposure</td>
      <td>&lt;securityType&gt;- Long Ratio</td>
      <td>Time series of the overall ratio of &lt;securityType&gt; long positions of the whole portfolio if any &lt;securityType&gt; is ever in the universe</td>
    </tr>
    <tr>
      <td>&lt;securityType&gt; - Short Ratio</td>
      <td>Time series of the overall ratio of &lt;securityType&gt; short position of the whole portfolio, if any &lt;securityType&gt; is ever in the universe</td>
    </tr>
```